src/Conversion/ONNXToKrnl/Math/LRN.cpp

```diff
@@ -72,14 +72,15 @@ struct ONNXLRNOpLowering : public ConversionPattern {
     Value cValue = outputLoops.getInductionVar(loopIndexForC);
     DimIndexExpr cIE(cValue);
     MemRefBoundIndexCapture inputBounds(input);
-    DimIndexExpr sizeIE(inputBounds.getDim(loopIndexForC));
+    DimIndexExpr CIE(inputBounds.getDim(loopIndexForC));
+    SymbolIndexExpr sizeIE = LiteralIndexExpr((int)sizeLit);
```

Comment:
Attribute size should be int. Could you change the initialization of sizeLit?

Reply:
Done. Thanks for your comment!